docs/data-hub/dataframes.md

```diff
@@ -0,0 +1,130 @@
+# Using Deep Origin DataFrames
+
+This page describes how to use Deep Origin DataFrames, which are the primary object you will use to interact with databases on Deep Origin. This page will cover:
```

Comment:
Should "which are the primary object" be "which are the primary objects" (object plural) or "which is the primary class" (class singular)?